src/components/PlayerUI.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect} from 'react';
+import WebPlayback from './WebPlayBack';
+
+//Acceptable format for props data.
+export interface IPlayerUIProps {
+    trackName: string,
+    albumName: string,
+    artistName: string
+}
+//@ts-ignore
```

Comment:
Do we still need this @ts-ignore?

Reply:
Unnecessary, was there when IPlayerUIProps didn't exist - instead there was a Variables being passed through props as any. Removed.